### ISSUE TITLE
backdrop pseudo element

### ISSUE THREAD

**Issue by thejackshelton** [oddbird/popover-polyfill#130]
Is it possible to add a backdrop polyfilled pseudo element maybe? I noticed there is a backdrop in the native html spec, and it works great in supported browsers, but looks like it doesn't in unsupported browsers like firefox.

**Comment by thagxt** [comment]
Yes, that would be nice to have. However, I ended up doing the following:

```css
.\:popover-open::after {
    content: '';
    position: fixed;
    inset: 0px;
    z-index: -1;
    pointer-events: none;
    background-color: red;
}
```

**Comment by thejackshelton** [comment]
> Yes, that would be nice to have. However, I ended up doing the following:
> 
> ```css
> .\:popover-open::after {
>     content: '';
>     position: fixed;
>     inset: 0px;
>     z-index: -1;
>     pointer-events: none;
>     background-color: red;
> }
> ```

It seems to me like the text goes in front of the backdrop, but the background of the popover doesn't. Are you experiencing the same? And if so any ideas? Trying to figure this out

**Comment by thagxt** [comment]
Yes, same here. I've also found other issues, I think they have to append a `div.backdrop` and only with CSS it might not be possible. For example, in some cases, because the `::after` is tied to the `.popover-open`, it does not take up the entire viewport but is confined to the size of the parent div.

However, I ended up doing some nesting:

```HTML
<div popover> <!-- I style this as if it is the backdrop -->
    <div class="wrap">
        <div class="content"></div>
    </div>
</div>
```

This approach also helps with the native element, allowing me to position `.wrap` wherever I want within the viewport using grid rather than inset. I'm using it for a modal, so it might not be ideal for other purposes.
